fdbcli/fdbcli.actor.cpp

```diff
@@ -989,6 +989,40 @@ void printStatus(StatusObjectReader statusObj, StatusClient::StatusLevel level,
 
 				if (statusObjConfig.get("log_routers", intVal))
 					outputString += format("\n  Desired Log Routers    - %d", intVal);
+
+				outputString += "\n  Usable Regions         - ";
+				if (statusObjConfig.get("usable_regions", intVal)) {
```

Comment:
Do you plan to add these new fields to
`Schemas.cpp` and `mr-status-json-schemas.rst.inc` (document), as the existing fields do?

Reply:
It’s already there. This change is to expose the existing JSON fields to the `status default|detail`